src/views/bav/confirm-details.html

```diff
@@ -0,0 +1,148 @@
+{% extends "base-form.njk" %}
+{# the content for this page is controlled by locales/en/default.yml #}
+{% set hmpoTitle = confirmDetails.title %}
+{% set hmpoPageKey = "confirmDetails" %}
+{% set gtmJourney = "bav - confirmDetails" %}
+
+{% from "govuk/components/header/macro.njk" import govukHeader %}
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
+{% from "hmpo-form/macro.njk" import hmpoForm %}
+
+{% block mainContent %}
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-three-quarters">
+    <h1 id="confirm-details-header" class="gov" data-page="{{hmpoPageKey}}" class="govuk-heading-l">
+        {{ translate("confirmDetails.title") }}
+    </h1>
+  </div>
+</div>
+
+{{ govukSummaryList({
+  classes: 'govuk-!-margin-bottom-6',
+  rows: [
+    {
+      key: {
+        text: ""
+      },
+      value: {
+        text: ""
```

Comment:
Is this just an empty row?

Reply:
Right so it's been designed differently to the standard govukSummaryList component that is being used across other repos? 

Leave it in for now, but this kind of thing is part of the reason I'd like us to start communicating more closely with designers - it's a really easy thing to miss if you're working in figma but it creates inconsistencies and tech debt